code/mob/living/silicon/ai.dm

```diff
@@ -1491,71 +1494,46 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
+	var/law_base = ""
+	if(law_base_choice == "Real Laws")
+		law_base = ticker.centralized_ai_laws.format_for_logs("\n")
+	else if(law_base_choice == "Fake Laws")
+		for(var/fake_law in src.fake_laws)
+			// this is just the default input for the user, so it should be fine
+			law_base += "[html_decode(fake_law)]\n"
+
+	var/raw_law_text = input(usr,"Please enter the fake laws you would like to be able to state via the State Fake Laws command! Each line is one law.", "Fake Laws", law_base) as null|message
+	if(!raw_law_text)
 		return
-	state = strip_html(state,MAX_MESSAGE_LEN)
-
-	var/list/laws_to_state = list()
-
-	laws_to_state = params2list(state)
-
-    // 1;1;1 becomes 1:list() and 1=2;1=3 becomes 1:(2,3), so we need to break the results down into one association per index
-
-	var/found = 0
-	for (var/index in laws_to_state)
-		if(laws_to_state[index])
-			for (var/association in laws_to_state[index])
-				if(association)
-					laws_to_state[index] = association
-					found = 1
-					break
-
-			if(!found)
-				laws_to_state[index] = null
-
-			found = 0
-
-	//build laws list from 0th, inherent, and supplied laws
-
-
-	var/list/laws_list = list()
-
-	var/number = 0
-	if(ticker.centralized_ai_laws.zeroth)
-		laws_list += "[number]"
-		laws_list["[number]"] = "[ticker.centralized_ai_laws.zeroth]"
-
-	number++
-
-	for (var/index = 1, index <= ticker.centralized_ai_laws.inherent.len, index++)
-		var/law = ticker.centralized_ai_laws.inherent[index]
-		if (length(law) > 0)
-			laws_list += "[number]"
-			laws_list["[number]"] += "[law]"
-			number++
-
-	for (var/index = 1, index <= ticker.centralized_ai_laws.supplied.len, index++)
-		var/law = ticker.centralized_ai_laws.supplied[index]
-		if (length(law) > 0)
-			laws_list += "[number]"
-			laws_list["[number]"] += "[law]"
-			number++
+	// split into lines
+	var/list/raw_law_list = splittext_char(raw_law_text, "\n")
+	// clear old fake laws
+	src.fake_laws = list()
+	// cleanse the lines and add them as our laws
+	for(var/raw_law in raw_law_list)
+		var/nice_law = trim(strip_html(raw_law))
+		// empty lines would probably be an accident and result in awkward pauses that might give the AI away
+		if (!length(nice_law))
+			continue
+		fake_laws += nice_law
 
-	//state laws in order given. Uses original numbers unless renumbering is specified
+	src.show_message("<span class='bold'>Your new fake laws are: </span>")
+	for(var/a_law in src.fake_laws)
+		src.show_message(a_law)
 
-	for(var/law_number in laws_to_state)
-		if(law_number in laws_list)
-			if(laws_to_state[law_number])
-				src.say("[laws_to_state[law_number]]. [laws_list[law_number]]")
-			else
-				src.say("[law_number]. [laws_list[law_number]]")
-			sleep(1 SECOND)
+/mob/living/silicon/ai/proc/ai_state_fake_laws()
+	set category = "AI Commands"
+	set name = "State Fake Laws"
 
+	for(var/a_law in src.fake_laws)
+		sleep(1 SECOND)
```

Comment:
would be nice if this delay was a define (to ensure this and the standard state laws verb were always in sync), but that's been an issue since way before this PR, so no worries

Reply:
True, I actually went and checked the other verbs to make sure this wasn't distinguishable by hearers, a define is a good idea!